FV3/configure

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -eu
+
```

Comment:
Ah I see, a consequence of us deciding to not run containers on daint, can't use the steps done in docker. This looks good to me!